executor/common_linux.h

```diff
@@ -150,7 +149,8 @@ static void netlink_attr(struct nlmsg* nlmsg, int typ,
 	struct nlattr* attr = (struct nlattr*)nlmsg->pos;
 	attr->nla_len = sizeof(*attr) + size;
 	attr->nla_type = typ;
-	memcpy(attr + 1, data, size);
+	if (size > 0)
```

Comment:
I think this as well could be a preparatory commit, just something like "allow zero-payload attribute changes ...."?

Reply:
Created a separate commit.